demos/tests/data_sequences.py

```diff
@@ -88,7 +88,6 @@
     ],
 
     'human-pose-estimation': [
-        image_net_arg('00000002'),
```

Comment:
What's the purpose of this change?

Reply:
This sequence consists of images with incorrect aspect ratio, which does not fit the model shape, so the error during model preprocessing occurs.

Reply:
OK.